### PR TITLE
Added security annotations to fix baseline scan

### PR DESCRIPTION
**Pull request by psiinon**
Latest baseline scan results can be see here: https://github.com/mozilla-services/foxsec/wiki/Baseline-services

### CODE REVIEW

File: src/olympia/bandwagon/templates/bandwagon/collection_detail.html [hunk 1]
```diff
@@ -179,7 +179,7 @@ <h4 class='author'>
     {% else %}
       <p>{% trans %}
         Add-ons that you mark as favorites using the <b>Add to Favorites</b> feature appear below.
-        This collection is currently <b>private</b>, which means only you can see it. If you would
+        This collection is currently <b>private</b>, which means only you can see it.  If you would
```
Comment (EnTeQuAk) [inline src/olympia/bandwagon/templates/bandwagon/collection_detail.html]: Whitespace :)

**Comment by psiinon** [comment]
My bad - fixed :)

**Comment by EnTeQuAk** [comment]
Looks good to me, do you have merge-permissions?

**Comment by psiinon** [comment]
I dont, no. Probably best that way ;)

**Pull request merged**